service/controller.go

```diff
@@ -28,6 +92,9 @@ func (s *service) CreateVolume(
 		fsType := mountCap.FsType
 		log.Info("FsType:", fsType)
 		fileResp, fileErr := FileCreateVolume(s, ctx, req)
+		if fileErr == nil {
```

Comment:
Do we have any `volume-existing` error? We should only catch the `volume-existing` error.

Reply:
I didn't touch the FileCreateVolume function, I just add the file volume prefix when the function returned successfully. I can look at that function next.